### PR TITLE
Resolve dependencies and fix demo errors by yarn

### PR DESCRIPTION
**Pull request by CitruXonve**
Solve the issue when dependencies like `yarn` and `parcel` are missing in a new developing environment, but required to run the demo locally. The `README` and `.gitignore` are thus updated accordingly.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h1>IPA-JFK</h1>
 You can obtain the source code from GitHub:
 https://github.com/b1f6c1c4/IPA-JFK</pre>
 
-    <script src="main.js"></script>
+    <script src="main.js" type="module"></script>
```
Comment (b1f6c1c4) [inline demo/index.html:60]: `demo/main.js` is NOT a JavaScript module. Why include `type="module"` here? :thinking:

File: README.md [hunk 2]
```diff
@@ -2,25 +2,43 @@
...
 [![Netlify Status](https://api.netlify.com/api/v1/badges/6cc76b39-d72c-4d05-9959-0d57d6b8b6e4/deploy-status)](https://app.netlify.com/sites/ipa-jfk/deploys)
 
 You can lookup words on our hassle-free **[Demo Website](https://jfk.b1f6c1c4.info/)**!
 
-## Install
+## Prerequisites
+
+- Node.js 12.x+
+- Yarn (recommended)
+- [Parcel](https://parceljs.org/getting_started.html) (for demo)
```
Comment (b1f6c1c4) [inline README.md:15]: You should point to `https://v2.parceljs.org/`

File: .gitignore [hunk 3]
```diff
@@ -112,6 +112,9 @@ dist
 # Stores VSCode versions used for testing VSCode extensions
 .vscode-test
 
+# Parcel cache
+.parcel-cache/
```
Comment (b1f6c1c4) [inline .gitignore:116]: `.npmignore` must be changed accordingly.

**Review by b1f6c1c4 — COMMENTED** [review]
BTW this project is not switching to yarn.

File: demo/index.html (hunk 1 above)
Comment (CitruXonve) [reply demo/index.html:60]: Was trying to import other libraries and `Eslint` panicked: `Parsing error: 'import' and 'export' may appear only with 'sourceType: module'`

File: README.md (hunk 2 above)
Comment (CitruXonve) [reply README.md:15]: Will do

File: .gitignore (hunk 3 above)
Comment (CitruXonve) [reply .gitignore:116]: Will do

**Comment by CitruXonve** [comment]
No explicit transition into `yarn` is needed though.

File: README.md (hunk 2 above)
Comment (CitruXonve) [reply README.md:15]: But beware of the migration tips [here](https://v2.parceljs.org/getting-started/migration/).

File: demo/index.html (hunk 1 above)
Comment (CitruXonve) [reply demo/index.html:60]: Update: migration from deprecated `parcel-bundler` to `parcel` requires `type="module"`. See also [Parcel 2 RC](https://v2.parceljs.org/blog/rc0/).

**Pull request closed without merging**